content/docs/reference-react-dom-server.md

```diff
@@ -61,15 +61,15 @@ If you plan to use React on the client to make the markup interactive, do not us
 ReactDOMServer.renderToNodeStream(element)
 ```
 
-Render a React element to its initial HTML. Returns a [Readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) that outputs an HTML string. The HTML output by this stream is exactly equal to what [`ReactDOMServer.renderToString`](#rendertostring) would return. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
+Renderiza um elemento React para seu HTML inicial. Retorna um [Stream legível](https://nodejs.org/api/stream.html#stream_readable_streams) que gera uma string HTML. A saída HTML desse stream é exatamente igual ao que [`ReactDOMServer.renderToString`](#rendertostring) retornaria. Você pode usar este método para gerar HTML no servidor e enviar o markup para o request inicial para ter carregamentos de página rápidos e para permitir que motores de pesquisa rastreiem suas páginas para fins de SEO.
 
-If you call [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on a node that already has this server-rendered markup, React will preserve it and only attach event handlers, allowing you to have a very performant first-load experience.
+Se você invocar [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) em um nó que já tem o seu markup processado pelo servidor, React vai preservá-lo e apenas atribuir manipuladores de eventos, permitindo que você tenha uma experiência de primeiro carregamento muito eficiente.
 
-> Note:
+> Nota:
 >
-> Server-only. This API is not available in the browser.
+> Apenas para servidor. Esta API não está disponível no navegador.
 >
-> The stream returned from this method will return a byte stream encoded in utf-8. If you need a stream in another encoding, take a look at a project like [iconv-lite](https://www.npmjs.com/package/iconv-lite), which provides transform streams for transcoding text.
+> O stream retornado deste método retornará um stream de bytes codificado em utf-8. Se você precisar de um stream em outra codificação, dê uma olhada em um projeto como o [iconv-lite](https://www.npmjs.com/package/iconv-lite), que fornece streams de transformação para transcodificação de texto.
```

Comment:
```suggestion
> A stream retornada deste método retornará uma stream de bytes codificado em utf-8. Se você precisar de uma stream em outra codificação, dê uma olhada em um projeto como o [iconv-lite](https://www.npmjs.com/package/iconv-lite), que fornece streams de transformação para transcodificação de texto.
```

Reply:
@koka0012 `stream` seria uma palavra no masculino. Logo seria `O stream`

Ex.: https://www.significados.com.br/streaming/ 

cc: @fjoshuajr @cezaraugusto @WendellAdriel

Reply:
I'll wait for consensus in the [Glossary and Style Guide](https://github.com/reactjs/pt-BR.reactjs.org/issues/4#issuecomment-461932897)

Reply:
I think it's better to have it with "o". not sure other places but at least Pt-BR and Pt-PT seems to use the masculine version